.github/workflows/maven.yml

```diff
@@ -14,8 +14,8 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest]
         java: [1.8, 11]
-        runtime: [ol]
-        runtime_version: [22.0.0.9-beta]
+        runtime: [ol, olbeta, wlp]
+        runtime_version: [22.0.0.8, 22.0.0.9-beta, 22.0.0.8]
```

Comment:
You'll need to setup some exclusions so that the `-beta` runtime_version only gets combined with the `olbeta` runtime, and the non `-beta` runtime_version only gets combined with the non-beta `ol` and `wlp` runtime.

Reply:
yeah it didn't work like I wanted it to.  I put it a different way, but if you have ideas on how to do it differently, I am all ears.

Reply:
What you did works, but if you want to make it simpler and have only one `mvn verify` invocation...you can add something like this in the matrix:

```
exclude:
        - runtime: olbeta
          runtime_version: 22.0.0.8
        - runtime: ol
          runtime_version: 22.0.0.9-beta
        - runtime: wlp
          runtime_version: 22.0.0.9-beta
```

Reply:
That would go right after the `runtime` and `runtime_version` lines which should look like:

```
      runtime: [ol, olbeta, wlp]
      runtime_version: [22.0.0.8, 22.0.0.9-beta]
```

Reply:
ok that seems to be working.  Thanks.